src/test/java/com/epam/atm/selenium/pageobject/NewApplicationFirstForm.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.atm.selenium.pageobject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+public class NewApplicationFirstForm extends AbstractPage {
+    private static final By TRADE_NAME_FIELD_LOCATOR = By.xpath("//input[@id='input-6']");
+    private static final By EMAIL_FIELD_LOCATOR = By.xpath("//input[@id=\'input-9\']");
```

Comment:
make this two locators with css just to train in css
+    private static final By TRADE_NAME_FIELD_LOCATOR = By.xpath("//input[@id='input-6']");
+    private static final By EMAIL_FIELD_LOCATOR = By.xpath("//input[@id=\'input-9\']");